src/ngAnimate/animate.js

```diff
@@ -1869,7 +1869,7 @@ angular.module('ngAnimate', ['ng'])
           return;
         }
 
-        if (!staggerTime && styles) {
+        if (!staggerTime && styles && Object.keys(styles).length > 0) {
```

Comment:
Oh I think it cannot since in `$animate.animate(element, from, to, ...)` `from` and `to` are both supposed to be objects.